datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyPartitionAssignmentStrategy.java

```diff
@@ -141,6 +141,116 @@ public StickyPartitionAssignmentStrategy(Optional<Integer> maxTasks, Optional<In
     return newAssignment;
   }
 
+  /**
+   * Move a partition for a datastream group according to the targetAssignment. As we are only allowed to mutate the
+   * task once. It follow the steps
+   * Step 1) get the partitions that to be moved, and get their source task
+   * Step 2) If the instance is the instance we want to move, we find out the task that we want to assign the task
```

Comment:
typo: we find out the task that we want to assign the topic partition?

Reply:
Fixed, thanks.